build.sh

```diff
@@ -29,6 +29,8 @@ PRINT_USAGE() {
     echo "  -j [N], --jobs[=N]   Multicore build, allow N jobs at once"
```

Comment:
Yeah I had to test my changes on linux and wanted some extra support. I can remove this part from the PR and open another one.